packages/fields-auto-increment/src/Implementation.js

```diff
@@ -47,6 +47,13 @@ export class AutoIncrementImplementation extends Implementation {
   gqlCreateInputFields() {
     return [`${this.path}: ${this.gqlType}`];
   }
+
+  getBackingTypes() {
+    if (this.path === 'id') {
+      return { [this.path]: { optional: false, type: 'number' } };
```

Comment:
I think this was wrong initially and should be string because a post read hook toStrings the id.
```suggestion
      return { [this.path]: { optional: false, type: 'string' } };
```